planner/languages/golang/golang_planner.go

```diff
@@ -42,13 +42,15 @@ func (p *Planner) GetPlan(srcDir string) *plansdk.Plan {
 		},
 		SharedPlan: plansdk.SharedPlan{
 			InstallStage: &plansdk.Stage{
-				Command: "go get",
+				InputFiles: []string{"."},
```

Comment:
This feels like it could be error prone. Thoughts on:

`plansdk.NewStageWithAllFiles`

or something like that? 

We could also do `plansdk.NewStage` copy all files by default. And we could do:

`plansdk.NewStage("cmd", plansdk.WithInputFiles(...))` to customize.

Reply:
I'm still not sure how to do it gracefully. We need the default of `[]string{"."}` for install and start stage, but not build stage. (Copying everything in build stage results in unexpected behavior. Probably not desired)

Reply:
@mikeland86 we use buildStage.inputFiles for `node_planner`

Reply:
got it! makes sense.

maybe we can add some syntactic sugar later.